workitem_whitebox_test.go

```diff
@@ -20,15 +22,25 @@ var DB *gorm.DB
 var rwiScheduler *remoteworkitem.Scheduler
 
 func TestMain(m *testing.M) {
+	var err error
+
+	if err = configuration.SetupConfiguration(configuration.DefaultConfigFilePath); err != nil {
+		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+	}
+
 	if _, c := os.LookupEnv(resource.Database); c != false {
```

Comment:
There are plans from others to have a simpler connection to the DB in tests (but I cannot find the PR/issue right now). For now I just replaced the configuration values with the ones from viper and left everything the way it is.